infra.go

```diff
@@ -70,8 +70,7 @@ type InfraInput struct {
 	Schema   string              `json:"schema" yaml:"schema" default:"Database"`
 	Owner    *ID                 `json:"owner" yaml:"owner" default:"XXX_owner_id_XXX"`
 	Provider *InfraProviderInput `json:"provider" yaml:"provider"`
-	// TODO: should InfraInput.Data be JSON type?
-	Data map[string]any `json:"data" yaml:"data" default:"{\"name\":\"my-big-query\",\"engine\":\"BigQuery\",\"endpoint\":\"https://google.com\",\"replica\":false}"`
+	Data     map[string]any      `json:"data" yaml:"data" default:"{\"name\":\"my-big-query\",\"engine\":\"BigQuery\",\"endpoint\":\"https://google.com\",\"replica\":false}"`
```

Comment:
@davidbloss this is a valid place where we could use `JSON` since the API is expecting a `JSON` type input, so feel free to re-add the comment above this line.

Reply:
I created [a ticket](https://github.com/orgs/OpsLevel/projects/1/views/1?pane=issue&itemId=47707462) to look into this instead :)